#477 Update workflow-example to 0.9.0

tortmayr opened the pull request:
Update the workflow example to 0.9.0. 
Source files have been copied from the 0.9.0 Release Dev example.
Copyright headers are only updated if the copied file has been modifed afterwards.

Also:
- Add gifs to main README
- Cleanup example READMEs
- Enable file logging if the examples are started in embedded server mode
- Add resolutions block to minimal example to avoid accidential resolving to incompatible Theia versions (>=1.21.0)

Part of eclipse-glsp/glsp/477

ndoschek commented on workflow/glsp-server/src/main/resources/glsp-graph.genmodel:
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <genmodel:GenModel xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:ecore="http://www.eclipse.org/emf/2002/Ecore"
-    xmlns:genmodel="http://www.eclipse.org/emf/2002/GenModel" copyrightText=" Copyright (c) 2019 EclipseSource and others.&#xA;&#xA;This program and the accompanying materials are made available under the&#xA;terms of the Eclipse Public License v. 2.0 which is available at&#xA;https://www.eclipse.org/legal/epl-2.0.&#xA;&#xA;This Source Code may also be made available under the following Secondary ...
+    xmlns:genmodel="http://www.eclipse.org/emf/2002/GenModel" copyrightText=" Copyright (c) 2019-2021 EclipseSource and others.&#xA;&#xA;This program and the accompanying materials are made available under the&#xA;terms of the Eclipse Public License v. 2.0 which is available at&#xA;https://www.eclipse.org/legal/epl-2.0.&#xA;&#xA;This Source Code may also be made available under the following Seco ...
```
```suggestion
    xmlns:genmodel="http://www.eclipse.org/emf/2002/GenModel" copyrightText=" Copyright (c) 2019-2022 EclipseSource and others.&#xA;&#xA;This program and the accompanying materials are made available under the&#xA;terms of the Eclipse Public License v. 2.0 which is available at&#xA;https://www.eclipse.org/legal/epl-2.0.&#xA;&#xA;This Source Code may also be made available under the following Secondary&#xA;Licenses when the conditions for such availability set forth in the Eclipse&#xA;Public License v. 2.0 are satisfied: GNU General Public License, version 2&#xA; with the GNU Classpath Exception which is available at&#xA; https://www.gnu.org/software/classpath/license.html.&#xA;&#xA;SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0&#xA; ********************************************************************************"
```

ndoschek commented on workflow/org.eclipse.glsp.graph/.project:
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Do we need that file?

ndoschek reviewed (changes requested):
Thanks Tobias, changes look great to me and the workflow example works as expected! :tada: 

I only have some minor nitpicks (please see inline) and one minor issue:
- Could you please update the VSCode task `Start Workflow GLSP Server` in `workflow/glsp-server` as it currently expects the 0.8.0*-jar.

tortmayr commented:
Thanks Nina, I addressed most of your review comments,
Regarding the copyright headers:
As mentioned in the PR description most files have just been replaced with their counterpart of the 0.9.0 GLSP release.
>Source files have been copied from the 0.9.0 Release Dev example.
Copyright headers are only updated if the copied file has been modifed afterwards.

I don't think it's good practice to adapt copyright headers of source files that have just been copied and are otherwise untouched. So I'd prefer to keep the copyright headers as is but if you disagree I'm also willing to update them (no strong opinion on that)

ndoschek reviewed (approved):
Thanks Tobias for the update! :+1: 
Regarding the headers: Sorry I overlooked your comment in the description, so I agree to leave it like that.
